packages/oid4vci-demo-frontend/src/configs/kvk.json

```diff
@@ -87,27 +87,24 @@
       }
     },
     "SSICredentialVerifyRequestPage": {
-      "enableRightPaneButton" : false,
```

Comment:
This has to stay, we do not need a button on this page

Reply:
we are checking for undefined